explainer.md

```diff
@@ -137,23 +137,23 @@ https://github.com/w3c/webrtc-pc/issues/2300
 https://github.com/w3c/webrtc-pc/issues/2309
  
 #### Abstract
-This document provides an API for enabling/disabling a sender in a real-time audio-video calling to adapt its audio packet rate for a better utilizing of the network connection between the sender and other participants.
+This document provides an API for enabling/disabling a sender in a real-time audio-video call to adapt its audio packet rate to better utilize the network connection between the sender and other participants.
 
 #### Introduction
-Congestion control is a common way for real-time audio-video conferencing to achieve a good performance [1], because, without it, the senders in a calling may send too much data and congest the network, and as a consequence, detriments the real time requirement of the calling.
+Congestion control is a common way for real-time audio-video conferencing to achieve a good performance [1], because, without it, the senders in a call may send too much data and congest the network, thus degrading call quality by increasing latency.
```

Comment:
Is "by increasing latency" the best phrase here? I'd expect packets being lost, which makes it hard to decode the stream on the other end and quality hurting as a result, and that the sender would try to adjust to the BW by sending less quality content at the end. Is latency really the relevant factor here?